test/config_parser_SUITE.erl

```diff
@@ -3034,6 +3025,8 @@ compare_values({auth_opts, _}, V1, V2) ->
     compare_unordered_lists(V1, V2, fun handle_auth_opt/2);
 compare_values(outgoing_pools, V1, V2) ->
     compare_unordered_lists(V1, V2, fun handle_conn_pool/2);
+compare_values({modules, _}, [{mod_extdisco, V1}], [{mod_extdisco, V2}]) ->
+    compare_ordered_lists(V1, V2, fun compare_unordered_lists/2);
```

Comment:
This applies to cases when we add optional fields and break their alphabetical order*, for instance:
```
V1 = [[{host,"stun1"},{type,stun},{password,"password"}]]
V2 = [[{host,"stun1"},{password,"password"},{type,stun}]]
```
Thanks to this clause, we can step one level deeper and sort both lists gently.

\* Appending the `V1` list with e.g. `{username, "user"}` instead of `password` doesn't apply here.